gemini/reasoning-engine/tutorial_google_maps_agent.ipynb

```diff
@@ -989,46 +989,82 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "f0a73c613b9f"
+        "id": "9f2f7d3ed7bd"
       },
       "source": [
-        "### Reusing your deployed agent from other applications or SDKs"
+        "### Querying your deployed agent\n",
+        "\n",
+        "You've now deployed your Reasoning Engine agent and can [interact with it in multiple ways](https://cloud.google.com/vertex-ai/generative-ai/docs/reasoning-engine/use), both within this notebook and from other applications or environments. The primary methods for accessing your deployed agent are via the Python client library or through REST API calls. Here's an overview of both methods:\n",
+        "\n",
+        "**Method 1: Reusing within this notebook or another Python environment**\n",
+        "\n",
+        "You can directly reuse and query the `remote_agent` instance you created in this notebook.\n",
+        "\n",
+        "Or, you can instantiate a new instance in another notebook or Python script. To do this, you'll need to retrieve your deployed agent's resource name that uniquely identifies your agent, which is a string that includes the project, location, and Reasoning Engine ID. You can retrieve it by running the following code in the notebook or environment where you created your agent:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 23,
+      "metadata": {
+        "id": "fdaf8b91413f"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "'projects/954731210883/locations/us-central1/reasoningEngines/223106848019812424'"
+            ]
+          },
+          "execution_count": 23,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
```

Comment:
I'd recommend removing this output since it includes a project number

Reply:
Done and removed project numbers in cc48b90. Thanks!